src/services/utilServices/__tests__/MailClient.spec.ts

```diff
@@ -24,10 +24,30 @@ const generateEmail = (recipient: string, subject: string, body: string) => ({
 
 describe("Mail Client", () => {
   afterEach(() => mockAxios.reset())
+  jest.useFakeTimers()
```

Comment:
Hey I am confused here, are we still intending to use `jest.useFakeTimers()`? or is this buggy to the point where we shouldnt use it? 
Sanity check here: should we use this after `sendMailResponse` rather than for all test methods?

Reply:
the CICD passes with the use of `jest.useFakeTimers()` here, do check in if this behaviour is still buggy ya

Reply:
Every test in `Users.spec.ts` involving MailClient except for 1 will freeze and timeout upon `jest.useFakeTimers()`.